lib/tasks/admin.rake

```diff
@@ -0,0 +1,15 @@
+namespace :admin do
```

Comment:
I sort of agree, but admin:promote_user is more descriptive in my opinion, and I think populate and depopulate are the only autolab tasks, so for something like this, which could be used in production, maybe a different namespace makes sense.